.travis.yml

```diff
@@ -1,28 +1,56 @@
-language: php
-
-php:
-  - 7.1
+language: bash
+dist: xenial
+stages:
+-  Tests
+-  Third party services dependent tests
```

Comment:
this stage is not used so i think whole stages: section could be removed
```yaml
jobs:
    include:
    -   stage: Tests
        script: echo 1
    -   script: echo 2
    -   stage: 3rd party
        script: echo 3
    -   stage: 3rd party
        script: echo 4
    -   stage: 3rd party
        script: echo 5
```

would create 2 stages without this section